aten/src/ATen/native/cuda/BinaryMiscOpsKernels.cu

```diff
@@ -1,10 +1,13 @@
+#include <ATen/AccumulateType.h>
```

Comment:
this doesn't seem right -- you don't need accumulate_type because there is no accumulation here.